Readme.md

```diff
@@ -12,10 +12,20 @@ Run the following commands to install:
 npm install apigee-apim -g
 ```
 
-Set environment variables (powershell)
+Set environment variables 
+
+```
+APIGEE_USERNAME=""
+APIGEE_PASSWORD=""
+```
+
+## proxy setup
+
+When behind a proxy setup the following environment variables.
+
 ```
-$env:APIGEE_USERNAME=""
-$env:APIGEE_PASSWORD=""
+proxy_host="example.com"
```

Comment:
should now be PROXY_HOST and PROXY_PORT

Reply:
resolved in [8cf6c6c](https://github.com/jnbalker/apigee-apim/commit/8cf6c6c47065964baba338a6af71825a4cbd2c1f)